Bindings/Java/Matlab/Utilities/opensimC3D.m

```diff
@@ -0,0 +1,132 @@
+classdef opensimC3D < matlab.mixin.SetGet
+% opensimC3D C3D data into OpenSim Tables
+%   Utility  Class that uses OpenSim's C3DFileAdapter() class to produce
+%   OpenSim tables. Methods include Simbody rotation.
+    properties (Access = private)
+    path
+    markers
+    forces
+    end
+    methods
+        function obj = opensimC3D(path2c3d)
+            % Class Constructor: input is an absolute path to a C3D file. 
+            
+            % verify the file path is correct
+            if exist(path2c3d, 'file') == 0
+                error('File does not exist. Check path is correct')
+            else
+                obj.path = path2c3d;
+            end
+            % load java libs
+            import org.opensim.modeling.*
+            % Use a c3dAdapter to turn read a c3d file
+            tables = C3DFileAdapter().read(path2c3d);
+            % get the marker and force data into OpenSim tables
+            obj.markers = tables.get('markers');
+            obj.forces = tables.get('forces');
+        end            
+        function rate = getRate_marker(obj)
+            % Get the capture rate used for the Marker Data
+            rate = str2double(obj.markers.getTableMetaDataAsString('DataRate'));
+        end
+        function rate = getRate_force(obj)
+            % Get the capture rate used for the Force Data
+            rate = str2double(obj.forces.getTableMetaDataAsString('DataRate'));
+        end
+        function n = getNumTrajectories(obj)
+            % Get the number of markers in the c3d file
+            n = obj.markers.getNumColumns;
```

Comment:
I think we should make sure to always use parentheses for functions, even if they take no arguments (`getNumColumns()`).